app/hotels/src/singleHotel/roomPicker/IncrementDecrement.js

```diff
@@ -0,0 +1,105 @@
+// @flow strict
+
+import * as React from 'react';
+import { View, Platform } from 'react-native';
+import {
+  Price,
+  Text,
+  StyleSheet,
+  IncrementDecrementButtons,
+  Color,
+} from '@kiwicom/mobile-shared';
+import { Translation } from '@kiwicom/mobile-localization';
+import { defaultTokens } from '@kiwicom/mobile-orbit';
+
+type Props = {|
+  +price: number | null,
+  +currency: string | null,
+  +selectedCount: number,
+  +selectableCount: number,
+  +increment: () => void,
+  +decrement: () => void,
+|};
+
+export default function IncrementDecrement({
+  decrement,
+  selectedCount,
+  price,
+  currency,
+  increment,
+  selectableCount,
+}: Props) {
+  return (
+    <View style={styles.container}>
+      <View style={styles.row}>
+        <View style={styles.priceContainer}>
+          <Text style={styles.countText}>
+            {Platform.select({
```

Comment:
Is it possible to make it clearer that only the prop textTransform is added on Android? Maybe creating the prop before the return statement and then just injecting or spreading a prop object?

Reply:
Could you show with an example, I am not sure what you mean 😄

Reply:
Of course.
So before `return`, have something like: 
```
const myAndroidProps = Platform.OS === 'ios' ? {} : {textTransform: 'uppercase'};
```

and then just write 
```
<Translation id="the.id" {...myAndroidProps} />
```